lms/djangoapps/grades/new/course_grade.py

```diff
@@ -26,6 +26,12 @@ def __init__(self, student, course, course_structure):
         self.course_structure = course_structure
         self.chapter_grades = []
 
+        # TODO: currently, data model won't accept null for course version, but tests do that a lot. Determine correct solution to this problem.
+            # - course_version not always needed for PersistentSubsectionGrade?
+            # - tests all need to ensure their courses have a course_version?
+        if self.course.course_version is None:
+            self.course.course_version = 1
```

Comment:
Not all courses will have a `course_version`.  Only courses created in `SplitModulestore` will have a value for this field.  Other courses (i.e., created in the Old Mongo Modulestore) will not have a value for this field.